packages/gatsby-remark-prismjs/highlight-line-range.js

```diff
@@ -0,0 +1,160 @@
+"use strict"
```

Comment:
This file shouldn't be committed! We'll want to remove

Reply:
oopsie, removed 😅